app/helpers/conservation_records_helper.rb

```diff
@@ -1,13 +1,6 @@
 # frozen_string_literal: true
 
 module ConservationRecordsHelper
-  def date_returned(record)
```

Comment:
@sameeraappana8 Some of your circleci specs are giving this error:
```
ActionView::Template::Error:
        undefined method `date_returned' for #<#<Class:0x0000557652c04478>:0x0000557656d761e0>
        Did you mean?  date_field
 ```

That's because your PR has removed the `date_returned` method.  Something might have happened in a rebase.  So go through all the changed files in your PR and make sure there are no weird changes.  I see other changes to  app/models/conservation_record.rb, app/views/conservation_records/show.html.erb, and some views that probably shouldn't be there.  Just put the code back the way it was and add the commit to your PR.